testing/tests/DevExpress.viz.core.series/lineSeries.tests.js

```diff
@@ -1284,6 +1290,44 @@ function setDiscreteType(series) {
         });
     });
 
+    QUnit.test('Custom styles should not apply, normal state', function(assert) {
+        const series = this.createSeries({ ...this.options, ...{ color: { fillId: 'id_color', base: 'n-color' } } });
```

Comment:
It's not required to create a new object and distructurize it, we can just specify new props => code is simpler and faster (desctucturization = O(n))
```suggestion
        const series = this.createSeries({ ...this.options, color: { fillId: 'id_color', base: 'n-color' } });
```